### PR TITLE
add yotpo as a source docs

### PR DESCRIPTION
**Pull request by vladimir-marinov-yotpo**
### Proposed changes

Adding new docs for the Yotpo as a source 

### Merge timing
ASAP once approved.

### CODE REVIEW

File: src/connections/sources/catalog/cloud-apps/yotpo/index.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+Yotpo Source
+---
```
Comment (forstisabella) [inline src/connections/sources/catalog/cloud-apps/yotpo/index.md:2]:
```suggestion
---
title: Yotpo Source
beta: true
---
```

File: src/connections/sources/catalog/cloud-apps/yotpo/index.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+[Yotpo](https://yotpo.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners) is a retention marketing platform that helps turn one-time shoppers into customers for life and maximize customer value. It enables you to collect and showcase customer reviews, offer loyalty programs and SMS and email marketing solutions.
```
Comment (forstisabella) [inline src/connections/sources/catalog/cloud-apps/yotpo/index.md:4]:
```suggestion
[Yotpo](https://yotpo.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners) {:target="_blank”} is a retention marketing platform that helps turn one-time shoppers into customers for life and maximize customer value. It enables you to collect and showcase customer reviews, offer loyalty programs and SMS and email marketing solutions.
```

File: src/connections/sources/catalog/cloud-apps/yotpo/index.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+Take your company’s customer engagement to the next level by adding Yotpo as a Source to Segment. Segment automatically collects engagement events like SMS sent, SMS clicked and Email opened, forwards them to your destinations, and loads them into your data warehouse.
```
Comment (forstisabella) [inline src/connections/sources/catalog/cloud-apps/yotpo/index.md:6]:
```suggestion
Take your company’s customer engagement to the next level by adding Yotpo as a Source to Segment. Segment automatically collects engagement events (like SMS sent, SMS clicked and Email opened) forwards them to your destinations, and loads them into your data warehouse.
```

File: src/connections/sources/catalog/cloud-apps/yotpo/index.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+## Getting started
+
```
Comment (forstisabella) [inline src/connections/sources/catalog/cloud-apps/yotpo/index.md:9]:
```suggestion
```

File: src/connections/sources/catalog/cloud-apps/yotpo/index.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+1. From your workspace's [Sources catalog page](https://app.segment.com/goto-my-workspace/sources/catalog) click **Add Source**.
+2. Search for **Yotpo** in the Sources Catalog, select Yotpo, and click **Add Source**.
+2. On the next screen, give the Source a name.
```
Comment (forstisabella) [inline src/connections/sources/catalog/cloud-apps/yotpo/index.md:13]:
```suggestion
2. On the next screen, give the Source a name. You can also apply labels to the source and connect the source to your warehouses. When you've finished setting up your source, click **Add Source**. 
```

File: src/connections/sources/catalog/cloud-apps/yotpo/index.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+3. The next page (“Overview”) will surface your Segment write key for Yotpo. Copy this write key.
```
Comment (forstisabella) [inline src/connections/sources/catalog/cloud-apps/yotpo/index.md:14]:
```suggestion
3. Segment redirects you to the Overview page and surfaces your Segment write key for Yotpo. Copy this write key.
```

File: src/connections/sources/catalog/cloud-apps/yotpo/index.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+4. In your Yotpo integration center, search for "Segment" and click on Connect.
```
Comment (forstisabella) [inline src/connections/sources/catalog/cloud-apps/yotpo/index.md:15]:
```suggestion
4. In your Yotpo integration center, search for "Segment" and click **Connect**.
```

File: src/connections/sources/catalog/cloud-apps/yotpo/index.md [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+5. On the Provide API Key stage of the integration process, enter the **Segment Write Key** in the right place.
+6. Click **Connect**.
+7. Go back to Segment and click **Add Destinations** in your Yotpo source to add the destinations where you want to see SMS and email events.
```
Comment (forstisabella) [inline src/connections/sources/catalog/cloud-apps/yotpo/index.md:18]:
```suggestion
7. Return to Segment and click **Add Destinations** in your Yotpo source to add the destinations where you want to send your SMS and email events.
```

File: src/connections/sources/catalog/cloud-apps/yotpo/index.md [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+| Email unsubscribed | User clicked the unsubscribe link in an email.           |
+| Email subscribed   | User subscribed to receive emails.                       |
+
+
+
+## Adding Destinations
+
+Now that your Source is set up, you can connect it with Destinations.
+
+Log into your downstream tools and check to see that your events appear as expected, and that they contain all of the properties you expect. If your events and properties don’t appear, check the Event Delivery tool, and refer to the Destination docs for each tool for troubleshooting.
```
Comment (forstisabella) [inline src/connections/sources/catalog/cloud-apps/yotpo/index.md:44]:
```suggestion
Log into your downstream tools and check to see that your events appear as expected, and that they contain all of the properties you expect. If your events and properties don’t appear, check the Event Delivery tool, and refer to the docs for each destination for troubleshooting.
```

File: src/connections/sources/catalog/cloud-apps/yotpo/index.md [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+| Email clicked      | User clicked a link within an email.                     |
+| Email bounced      | An email could not be delivered to the user.             |
+| Email unsubscribed | User clicked the unsubscribe link in an email.           |
+| Email subscribed   | User subscribed to receive emails.                       |
+
+
+
+## Adding Destinations
+
+Now that your Source is set up, you can connect it with Destinations.
```
Comment (forstisabella) [inline src/connections/sources/catalog/cloud-apps/yotpo/index.md:42]:
```suggestion
Now that your source is set up, you can connect it to destinations.
```

File: src/connections/sources/catalog/cloud-apps/yotpo/index.md [hunk 11]
```diff
@@ -0,0 +1,46 @@
...
+2. On the next screen, give the Source a name.
+3. The next page (“Overview”) will surface your Segment write key for Yotpo. Copy this write key.
+4. In your Yotpo integration center, search for "Segment" and click on Connect.
+5. On the Provide API Key stage of the integration process, enter the **Segment Write Key** in the right place.
+6. Click **Connect**.
+7. Go back to Segment and click **Add Destinations** in your Yotpo source to add the destinations where you want to see SMS and email events.
+
+## Events
+
+The table below lists events that Yotpo sends to Segment. These events appear as tables in your warehouse, and as regular events in other Destinations. Yotpo includes the `userId` if available.
```
Comment (forstisabella) [inline src/connections/sources/catalog/cloud-apps/yotpo/index.md:22]:
```suggestion
The table below lists events that Yotpo sends to Segment. These events appear as tables in your warehouse, and as regular events in other destinations. Yotpo includes the `userId` if available.
```

File: src/connections/sources/catalog/cloud-apps/yotpo/index.md [hunk 12]
```diff
@@ -0,0 +1,46 @@
...
+---
+
+[Yotpo](https://yotpo.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners) is a retention marketing platform that helps turn one-time shoppers into customers for life and maximize customer value. It enables you to collect and showcase customer reviews, offer loyalty programs and SMS and email marketing solutions.
+
+Take your company’s customer engagement to the next level by adding Yotpo as a Source to Segment. Segment automatically collects engagement events like SMS sent, SMS clicked and Email opened, forwards them to your destinations, and loads them into your data warehouse.
+
+## Getting started
+
+
+1. From your workspace's [Sources catalog page](https://app.segment.com/goto-my-workspace/sources/catalog) click **Add Source**.
```
Comment (forstisabella) [inline src/connections/sources/catalog/cloud-apps/yotpo/index.md:11]:
```suggestion
1. From your workspace's [Sources catalog page](https://app.segment.com/goto-my-workspace/sources/catalog) click **Add source**.
```

**Comment by vladimir-marinov-yotpo** [comment]
@forstisabella, I have reviewed and applied the suggested comments. Feel free to let me know if there is anything else you need me to do.

File: src/connections/sources/catalog/cloud-apps/yotpo/index.md [hunk 13]
```diff
@@ -0,0 +1,47 @@
+---
+title: Yotpo Source
+beta: true
+---
```
Comment (tcgilbert) [inline src/connections/sources/catalog/cloud-apps/yotpo/index.md:4]:
```suggestion
---
title: Yotpo Source
beta: true
id: q4JbVJwmrg
---
```

File: src/connections/sources/catalog/cloud-apps/yotpo/index.md [hunk 14]
```diff
@@ -0,0 +1,47 @@
...
+2. On the next screen, give the Source a name. You can also apply labels to the source and connect the source to your warehouses. When you've finished setting up your source, click **Add Source**. 
+3. Segment redirects you to the Overview page and surfaces your Segment write key for Yotpo. Copy this write key.
+4. In your Yotpo integration center, search for "Segment" and click **Connect**.
+5. On the Provide API Key stage of the integration process, enter the **Segment Write Key** in the right place.
+6. Click **Connect**.
+7. Return to Segment and click **Add Destinations** in your Yotpo source to add the destinations where you want to send your SMS and email events.
+
+## Events
+
+The table below lists events that Yotpo sends to Segment. These events appear as tables in your warehouse, and as regular events in other destinations. Yotpo includes the `userId` if available.
```
Comment (tcgilbert) [inline src/connections/sources/catalog/cloud-apps/yotpo/index.md:23]: > The table below lists events that Yotpo sends to Segment. These events appear as tables in your warehouse, and as regular events in other destinations. Yotpo includes the `userId` if available.

- Could you explain what identifier is used for the Segment `userId` value (is it a yotpo id, or a user id that comes from customer)? 
- If there is no `userId` available, will you send data with an Segment `anonymousId`? If so, what will be used for `anonymousId`?

File: src/connections/sources/catalog/cloud-apps/yotpo/index.md [hunk 15]
```diff
@@ -0,0 +1,47 @@
...
+| SMS clicked        | User clicked a short link in an SMS message.             | 
+| SMS unsubscribed   | User clicked the unsubscribe link in an SMS.             | 
+| SMS subscribed     | User subscribed to receive SMS messages.                 |
+| Email sent         | An email was sent to the user.                           |
+| Email delivered    | An email was successfully delivered to the user's inbox. |
+| Email opened       | User opened an email.                                    |
+| Email clicked      | User clicked a link within an email.                     |
+| Email bounced      | An email could not be delivered to the user.             |
+| Email unsubscribed | User clicked the unsubscribe link in an email.           |
+| Email subscribed   | User subscribed to receive emails.                       |
```
Comment (tcgilbert) [inline src/connections/sources/catalog/cloud-apps/yotpo/index.md:37]:
```suggestion
| SMS Sent           | An SMS was sent.                                         |
| SMS Clicked        | User clicked a short link in an SMS message.             | 
| SMS Unsubscribed   | User clicked the unsubscribe link in an SMS.             | 
| SMS Subscribed     | User subscribed to receive SMS messages.                 |
| Email Sent         | An email was sent to the user.                           |
| Email Delivered    | An email was successfully delivered to the user's inbox. |
| Email Opened       | User opened an email.                                    |
| Email Clicked      | User clicked a link within an email.                     |
| Email Bounced      | An email could not be delivered to the user.             |
| Email Unsubscribed | User clicked the unsubscribe link in an email.           |
| Email Subscribed   | User subscribed to receive emails.                       |
```

Per our event naming conventions, please convert your event names to `Proper Casing`: https://segment.com/docs/partners/sources/#naming-conventions

File: src/connections/sources/catalog/cloud-apps/yotpo/index.md [hunk 16]
```diff
@@ -0,0 +1,47 @@
...
+
+
```
Comment (tcgilbert) [inline src/connections/sources/catalog/cloud-apps/yotpo/index.md:39]: Please add a table outlining the properties that can be expected to be in these events: 

```
## Event Properties

The table below list the properties included in the events listed above.

| Property Name   | Description               |
| --------------- | ------------------------- |
| `email_id`      | ID of the email           |
| `from_id`        | Sender email ID           |
| `email_subject` | Subject line of the email |
| `link`          | URL of the link clicked   |
etc...
```

**Review by tcgilbert — CHANGES_REQUESTED** [review]

File: src/connections/sources/catalog/cloud-apps/yotpo/index.md (hunk 14 above)
Comment (vladimir-marinov-yotpo) [reply src/connections/sources/catalog/cloud-apps/yotpo/index.md:24]: Hi, 
`UserId` will be an id that comes from the customer. If that is not present, we don't send it.
We always send the `AnonymousId`, which is either a phone number or email based on the type of the event.

File: src/connections/sources/catalog/cloud-apps/yotpo/index.md (hunk 16 above)
Comment (vladimir-marinov-yotpo) [reply src/connections/sources/catalog/cloud-apps/yotpo/index.md:40]: Sure.

Comment (vladimir-marinov-yotpo) [reply src/connections/sources/catalog/cloud-apps/yotpo/index.md:40]: I am waiting for the descriptions to be approved, and I will update the PR.

Comment (vladimir-marinov-yotpo) [reply src/connections/sources/catalog/cloud-apps/yotpo/index.md]: Done.

**Comment by vladimir-marinov-yotpo** [comment]
@tcgilbert could you recheck the PR?

File: src/connections/sources/catalog/cloud-apps/yotpo/index.md [hunk 17]
```diff
@@ -0,0 +1,61 @@
...
+2. On the next screen, give the Source a name. You can also apply labels to the source and connect the source to your warehouses. When you've finished setting up your source, click **Add Source**. 
+3. Segment redirects you to the Overview page and surfaces your Segment write key for Yotpo. Copy this write key.
+4. In your Yotpo integration center, search for "Segment" and click **Connect**.
+5. On the Provide API Key stage of the integration process, enter the **Segment Write Key** in the right place.
+6. Click **Connect**.
+7. Return to Segment and click **Add Destinations** in your Yotpo source to add the destinations where you want to send your SMS and email events.
+
+## Events
+
+The table below lists events that Yotpo sends to Segment. These events appear as tables in your warehouse, and as regular events in other destinations. Yotpo includes the `userId` if available.
```
[the remark this answers is not in the document]
Comment (tcgilbert) [reply src/connections/sources/catalog/cloud-apps/yotpo/index.md:24]: @vladimir-marinov-yotpo one more change, and then I will approve ^

[the remark this answers is not in the document]
Comment (vladimir-marinov-yotpo) [reply src/connections/sources/catalog/cloud-apps/yotpo/index.md]: Done.

**Comment by tcgilbert** [comment]
@vladimir-marinov-yotpo I've approved the documentation. To get the source live in the catalog, please navigate to the dev portal and submit the source for final review: 

https://app.segment.com/dev-portal/integrations/source/yotpo/status
Screenshot 2024-06-12 at 10 22 45 AM

File: src/connections/sources/catalog/cloud-apps/yotpo/index.md [hunk 18]
```diff
@@ -0,0 +1,61 @@
...
+Take your company’s customer engagement to the next level by adding Yotpo as a Source to Segment. Segment automatically collects engagement events (like SMS sent, SMS clicked, and Email opened) forwards them to your destinations, and loads them into your data warehouse.
+
+## Getting started
+
+1. From your workspace's [Sources catalog page](https://app.segment.com/goto-my-workspace/sources/catalog) click **Add source**.
+2. Search for **Yotpo** in the Sources Catalog, select Yotpo, and click **Add Source**.
+2. On the next screen, give the Source a name. You can also apply labels to the source and connect the source to your warehouses. When you've finished setting up your source, click **Add Source**. 
+3. Segment redirects you to the Overview page and surfaces your Segment write key for Yotpo. Copy this write key.
+4. In your Yotpo integration center, search for "Segment" and click **Connect**.
+5. On the Provide API Key stage of the integration process, enter the **Segment Write Key** in the right place.
```
Comment (forstisabella) [inline src/connections/sources/catalog/cloud-apps/yotpo/index.md:18]:
```suggestion
5. On the Provide API Key stage of the integration process, enter the **Segment Write Key**.
```

File: src/connections/sources/catalog/cloud-apps/yotpo/index.md [hunk 19]
```diff
@@ -0,0 +1,61 @@
...
+id: q4JbVJwmrg
+---
+
+[Yotpo](https://yotpo.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners) {:target="_blank”} is a retention marketing platform that helps turn one-time shoppers into customers for life and maximize customer value. It enables you to collect and showcase customer reviews, offer loyalty programs and SMS and email marketing solutions.
+
+Take your company’s customer engagement to the next level by adding Yotpo as a Source to Segment. Segment automatically collects engagement events (like SMS sent, SMS clicked, and Email opened) forwards them to your destinations, and loads them into your data warehouse.
+
+## Getting started
+
+1. From your workspace's [Sources catalog page](https://app.segment.com/goto-my-workspace/sources/catalog) click **Add source**.
```
Comment (forstisabella) [inline src/connections/sources/catalog/cloud-apps/yotpo/index.md:13]:
```suggestion
1. From your workspace's [Sources catalog page](https://app.segment.com/goto-my-workspace/sources/catalog){:target="_blank”} click **Add source**.
```

File: src/connections/sources/catalog/cloud-apps/yotpo/index.md [hunk 20]
```diff
@@ -0,0 +1,61 @@
+---
+title: Yotpo Source
+beta: true
+id: q4JbVJwmrg
+---
+
+[Yotpo](https://yotpo.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners) {:target="_blank”} is a retention marketing platform that helps turn one-time shoppers into customers for life and maximize customer value. It enables you to collect and showcase customer reviews, offer loyalty programs and SMS and email marketing solutions.
+
+Take your company’s customer engagement to the next level by adding Yotpo as a Source to Segment. Segment automatically collects engagement events (like SMS sent, SMS clicked, and Email opened) forwards them to your destinations, and loads them into your data warehouse.
```
Comment (forstisabella) [inline src/connections/sources/catalog/cloud-apps/yotpo/index.md:9]:
```suggestion
Take your company’s customer engagement to the next level by adding Yotpo as a Source to Segment. Segment automatically collects engagement events (like SMS sent, SMS clicked, and Email opened), forwards them to your destinations, and loads them into your data warehouse.
```

File: src/connections/sources/catalog/cloud-apps/yotpo/index.md [hunk 21]
```diff
@@ -0,0 +1,61 @@
+---
+title: Yotpo Source
+beta: true
+id: q4JbVJwmrg
+---
+
+[Yotpo](https://yotpo.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners) {:target="_blank”} is a retention marketing platform that helps turn one-time shoppers into customers for life and maximize customer value. It enables you to collect and showcase customer reviews, offer loyalty programs and SMS and email marketing solutions.
```
Comment (forstisabella) [inline src/connections/sources/catalog/cloud-apps/yotpo/index.md:7]:
```suggestion
[Yotpo](https://yotpo.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank”} is a retention marketing platform that helps turn one-time shoppers into customers for life and maximize customer value. It enables you to collect and showcase customer reviews, offer loyalty programs and SMS and email marketing solutions.
```

**Review by forstisabella — APPROVED** [review]

**Pull request merged**